### DISCUSSION TITLE
Do not sync while there is another session running.

### DISCUSSION THREAD

**Discussion by nallebrean** [remotely-save/remotely-save#591] [General]
When there is a running sync session and a new automatic sync starts there is a irritating warning messages about that.

It mostly occur during start up due to sync att start up and sync every minute. 

Suggested solutions might be that the error only shows up after some falied syncing sessions to alert the user of a real problem.

**Comment by fyears** [comment]
Have you updated to the latest version?

In the latest version, if a sync in running, another auto sync will not trigger the warning.

**Reply by nallebrean** [reply]
Yes it work in latest version!

Fantastic!

**Comment by nallebrean** [comment]
Closing
